ext/standard/formatted_print.c

```diff
@@ -450,7 +450,7 @@ php_formatted_print(zval *z_format, zval *args, int argc)
 
 					if (argnum <= 0) {
 						zend_string_efree(result);
-						php_error_docref(NULL, E_WARNING, "Argument number must be greater than zero");
+						zend_throw_error(NULL, "Argument number must be greater than zero");
```

Comment:
Please use Zend value error function i.e. ``zend_value_error("Argument number must be greater than zero");``